bigquery/snippets/snippet.go

```diff
@@ -407,6 +453,194 @@ func createTableExplicitSchema(client *bigquery.Client, datasetID, tableID strin
 	return nil
 }
 
+func createTableComplexSchema(client *bigquery.Client, datasetID, tableID string) error {
+	ctx := context.Background()
+	// [START bigquery_nested_repeated_schema]
+	sampleSchema := bigquery.Schema{
+		{Name: "id", Type: bigquery.StringFieldType},
+		{Name: "first_name", Type: bigquery.StringFieldType},
+		{Name: "last_name", Type: bigquery.StringFieldType},
+		{Name: "dob", Type: bigquery.DateFieldType},
+		{Name: "addresses",
+			Type:     bigquery.RecordFieldType,
+			Repeated: true,
+			Schema: bigquery.Schema{
+				{Name: "status", Type: bigquery.StringFieldType},
+				{Name: "address", Type: bigquery.StringFieldType},
+				{Name: "city", Type: bigquery.StringFieldType},
+				{Name: "state", Type: bigquery.StringFieldType},
+				{Name: "zip", Type: bigquery.StringFieldType},
+				{Name: "numberOfYears", Type: bigquery.StringFieldType},
+			}},
+	}
+
+	metaData := &bigquery.TableMetadata{
+		Schema:         sampleSchema,
+		ExpirationTime: time.Now().AddDate(1, 0, 0), // Table will be automatically deleted in 1 year.
```

Comment:
Is the expiration necessary for this sample? It seems unrelated to the actual schema demonstration.

Reply:
Will yank it.